src/olympia/addons/tasks.py

```diff
@@ -808,3 +809,27 @@ def disable_legacy_files(ids, **kw):
             for file_ in files:
                 log.info('Disabling file %d from addon %s', file_.pk, addon.pk)
                 file_.update(status=amo.STATUS_DISABLED)
+
+
+@task
+@use_primary_db
+def remove_amo_links_in_url_fields(ids, **kw):
+    """With the specified ids, remove the AMO links in the following URL fields
+    of each add-on: homepage, support_url, contribution."""
+    log.info('Deleting AMO links in URL fields %d-%d [%d].', ids[0], ids[-1],
+             len(ids))
+    addons = Addon.objects.filter(id__in=ids)
+    for addon in addons:
+        with transaction.atomic():
+            translation_ids = []
+            if addon.homepage:
+                translation_ids.append(addon.homepage.id)
+            if addon.support_url:
+                translation_ids.append(addon.support_url.id)
+            if translation_ids:
+                Translation.objects.filter(
+                    id__in=translation_ids,
+                    localized_string__icontains=settings.DOMAIN
+                ).update(localized_string=u'')
```

Comment:
you should update `localized_string_clean` to `''` too (it contains the `bleach`ed, escaped, etc version of the string)